### PR TITLE
Enable more clangarm64 packages

### PR DESCRIPTION
**Pull request by driver1998**
In preparation of VLC on CLANGARM64.

### CODE REVIEW

File: mingw-w64-SDL_image/PKGBUILD [hunk 1]
```diff
@@ -20,10 +20,21 @@ makedepends=("${MINGW_PACKAGE_PREFIX}-cc" "${MINGW_PACKAGE_PREFIX}-autotools")
 source=("${url}/release/${_realname}-${pkgver}.tar.gz")
 sha256sums=('0b90722984561004de84847744d566809dbb9daf732a9e503b91a1b5a84e5699')
 
+prepare() {
+  cd ${_realname}-${pkgver}
+
+  aclocal -I m4
+  libtoolize --copy --force
+  automake --foreign --add-missing --force-missing
+  autoconf
```
Comment (Biswa96) [inline mingw-w64-SDL_image/PKGBUILD:29]: Can this be replaced with `./autogen.sh`? Just 4 vs. 1 line.

Comment (driver1998) [reply mingw-w64-SDL_image/PKGBUILD:29]: after `autogen.sh`, `configure` errors with:
```
checking build system type... Invalid configuration `aarch64-w64-mingw32': machine `aarch64-w64' not recognized
```

`autoreconf -fiv` fails with:
```
autoreconf-2.71: export WARNINGS=
autoreconf-2.71: warning: autoconf input should be named 'configure.ac', not 'configure.in'
autoreconf-2.71: Entering directory '.'
autoreconf-2.71: configure.in: not using Gettext
autoreconf-2.71: running: aclocal --force
am-wrapper: aclocal: warning: auto-detected versions not found (1.10); falling back to latest available
aclocal-1.16: warning: autoconf input should be named 'configure.ac', not 'configure.in'
autoreconf-2.71: configure.in: tracing
autoreconf-2.71: running: libtoolize --copy --force
libtoolize: putting auxiliary files in '.'.
libtoolize: copying file './ltmain.sh'
libtoolize: Consider adding 'AC_CONFIG_MACRO_DIRS([m4])' to configure.in,
libtoolize: and rerunning libtoolize and aclocal.
libtoolize: Consider adding '-I m4' to ACLOCAL_AMFLAGS in Makefile.am.
autoreconf-2.71: configure.in: not using Intltool
autoreconf-2.71: configure.in: not using Gtkdoc
autoreconf-2.71: running: aclocal --force
aclocal-1.16: warning: autoconf input should be named 'configure.ac', not 'configure.in'
autoreconf-2.71: running: /usr/bin/autoconf-2.71 --force
configure.in:45: warning: AM_INIT_AUTOMAKE: two- and three-arguments forms are deprecated.
../autoconf-2.71/lib/autoconf/general.m4:2434: AC_DIAGNOSE is expanded from...
aclocal.m4:9985: AM_INIT_AUTOMAKE is expanded from...
configure.in:45: the top level
configure.in:48: warning: The macro `AC_PROG_LIBTOOL' is obsolete.
configure.in:48: You should run autoupdate.
aclocal.m4:121: AC_PROG_LIBTOOL is expanded from...
configure.in:48: the top level
configure.in:80: warning: The macro `AC_HELP_STRING' is obsolete.
configure.in:80: You should run autoupdate.
../autoconf-2.71/lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
../autoconf-2.71/lib/autoconf/general.m4:1534: AC_ARG_ENABLE is expanded from...
configure.in:80: the top level
configure.in:127: warning: The macro `AC_HELP_STRING' is obsolete.
configure.in:127: You should run autoupdate.
../autoconf-2.71/lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
../autoconf-2.71/lib/autoconf/general.m4:1534: AC_ARG_ENABLE is expanded from...
configure.in:127: the top level
configure.in:129: warning: The macro `AC_HELP_STRING' is obsolete.
configure.in:129: You should run autoupdate.
../autoconf-2.71/lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
../autoconf-2.71/lib/autoconf/general.m4:1534: AC_ARG_ENABLE is expanded from...
configure.in:129: the top level
configure.in:131: warning: The macro `AC_HELP_STRING' is obsolete.
configure.in:131: You should run autoupdate.
../autoconf-2.71/lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
../autoconf-2.71/lib/autoconf/general.m4:1534: AC_ARG_ENABLE is expanded from...
configure.in:131: the top level
configure.in:133: warning: The macro `AC_HELP_STRING' is obsolete.
configure.in:133: You should run autoupdate.
../autoconf-2.71/lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.in:133: the top level
configure.in:135: warning: The macro `AC_HELP_STRING' is obsolete.
configure.in:135: You should run autoupdate.
../autoconf-2.71/lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
../autoconf-2.71/lib/autoconf/general.m4:1534: AC_ARG_ENABLE is expanded from...
configure.in:135: the top level
configure.in:137: warning: The macro `AC_HELP_STRING' is obsolete.
configure.in:137: You should run autoupdate.
../autoconf-2.71/lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
../autoconf-2.71/lib/autoconf/general.m4:1534: AC_ARG_ENABLE is expanded from...
configure.in:137: the top level
configure.in:139: warning: The macro `AC_HELP_STRING' is obsolete.
configure.in:139: You should run autoupdate.
../autoconf-2.71/lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
../autoconf-2.71/lib/autoconf/general.m4:1534: AC_ARG_ENABLE is expanded from...
configure.in:139: the top level
configure.in:141: warning: The macro `AC_HELP_STRING' is obsolete.
configure.in:141: You should run autoupdate.
../autoconf-2.71/lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.in:141: the top level
configure.in:143: warning: The macro `AC_HELP_STRING' is obsolete.
configure.in:143: You should run autoupdate.
../autoconf-2.71/lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
../autoconf-2.71/lib/autoconf/general.m4:1534: AC_ARG_ENABLE is expanded from...
configure.in:143: the top level
configure.in:145: warning: The macro `AC_HELP_STRING' is obsolete.
configure.in:145: You should run autoupdate.
../autoconf-2.71/lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
../autoconf-2.71/lib/autoconf/general.m4:1534: AC_ARG_ENABLE is expanded from...
configure.in:145: the top level
configure.in:147: warning: The macro `AC_HELP_STRING' is obsolete.
configure.in:147: You should run autoupdate.
../autoconf-2.71/lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
../autoconf-2.71/lib/autoconf/general.m4:1534: AC_ARG_ENABLE is expanded from...
configure.in:147: the top level
configure.in:149: warning: The macro `AC_HELP_STRING' is obsolete.
configure.in:149: You should run autoupdate.
../autoconf-2.71/lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.in:149: the top level
configure.in:151: warning: The macro `AC_HELP_STRING' is obsolete.
configure.in:151: You should run autoupdate.
../autoconf-2.71/lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
../autoconf-2.71/lib/autoconf/general.m4:1534: AC_ARG_ENABLE is expanded from...
configure.in:151: the top level
configure.in:153: warning: The macro `AC_HELP_STRING' is obsolete.
configure.in:153: You should run autoupdate.
../autoconf-2.71/lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
../autoconf-2.71/lib/autoconf/general.m4:1534: AC_ARG_ENABLE is expanded from...
configure.in:153: the top level
configure.in:155: warning: The macro `AC_HELP_STRING' is obsolete.
configure.in:155: You should run autoupdate.
../autoconf-2.71/lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
../autoconf-2.71/lib/autoconf/general.m4:1534: AC_ARG_ENABLE is expanded from...
configure.in:155: the top level
configure.in:157: warning: The macro `AC_HELP_STRING' is obsolete.
configure.in:157: You should run autoupdate.
../autoconf-2.71/lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
../autoconf-2.71/lib/autoconf/general.m4:1534: AC_ARG_ENABLE is expanded from...
configure.in:157: the top level
configure.in:159: warning: The macro `AC_HELP_STRING' is obsolete.
configure.in:159: You should run autoupdate.
../autoconf-2.71/lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.in:159: the top level
configure.in:378: warning: AC_OUTPUT should be used without arguments.
configure.in:378: You should run autoupdate.
autoreconf-2.71: configure.in: not using Autoheader
autoreconf-2.71: running: automake --add-missing --copy --force-missing
automake-1.16: warning: autoconf input should be named 'configure.ac', not 'configure.in'
configure.in:45: warning: AM_INIT_AUTOMAKE: two- and three-arguments forms are deprecated.  For more info, see:
configure.in:45: https://www.gnu.org/software/automake/manual/automake.html#Modernize-AM_005fINIT_005fAUTOMAKE-invocation
configure.in:29: installing './compile'
configure.in:45: installing './missing'
Makefile.am:64: warning: '%'-style pattern rules are a GNU make extension
Makefile.am: error: required file './AUTHORS' not found
Makefile.am: error: required file './ChangeLog' not found
Makefile.am: installing './INSTALL'
Makefile.am: error: required file './NEWS' not found
automake-1.16: warning: autoconf input should be named 'configure.ac', not 'configure.in'
Makefile.am: installing './depcomp'
autoreconf-2.71: error: automake failed with exit status: 1
```

**Comment by Biswa96** [comment]
Ready to merge?

**Comment by driver1998** [comment]
Yeah.
VLC update will come later in separate PR.

**Pull request merged**